app/src/main/java/fr/iut_rodez/pathpilot_android_client/home/clients/FragmentClients.java

```diff
@@ -0,0 +1,66 @@
+package fr.iut_rodez.pathpilot_android_client.home.clients;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ListView;
+
+import androidx.fragment.app.Fragment;
+
+import java.util.List;
+
+import fr.iut_rodez.pathpilot_android_client.R;
+import fr.iut_rodez.pathpilot_android_client.home.Home;
+import fr.iut_rodez.pathpilot_android_client.model.Client;
+
+public class FragmentClients extends Fragment {
```

Comment:
Might be good to add class javadoc comment

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994